lib/transbank/sdk/utils/net_helper.rb

```diff
@@ -21,7 +21,7 @@ def http_post(uri_string:, body: nil, headers: nil, camel_case_keys: true)
         request_headers = {'Content-Type' => 'application/json'}.merge(headers || {})
         request = Net::HTTP::Post.new(uri.path, request_headers)
         sendable_body = camel_case_keys ? keys_to_camel_case(body) : body
-        request.body = JSON.generate(sendable_body)
+        request.body = sendable_body.to_json
```

Comment:
If you change this, It'd be good to change it on the rest of methods in this module that use JSON.generate

Reply:
I'll change it successively in the next transactions